src/service/controllers/voice.js

```diff
@@ -263,7 +265,7 @@ exports.handleCall = (req, res) => {
         req.body.data.destination
       );
       consultCall.parentConnId = call.id; //associate consult with the parent call
-      exports.makeCall(consultCall); //make the Consult call
+      exports.makeCall(consultCall, false); //make the Consult call
```

Comment:
I would like to understand more. If 2 different agents are logged into the simulator then will the consult call be delivered to the 'other' agent? if so then we can probably avoid the auto answer only in cases where we have a second agent. If not we can continue with auto answer.

Reply:
Consult calls cannot be made to agents that are offline. Hence there is no requirement for auto answer. In scenarios where the other agent is present, auto answer has been removed.

Reply:
ok